sdk/python/test_data/components/component_with_pip_install.yaml

```diff
@@ -19,7 +19,7 @@ deploymentSpec:
           \ python3 -m ensurepip --user || apt-get install python3-pip\nfi\n\nPIP_DISABLE_PIP_VERSION_CHECK=1\
           \ python3 -m pip install --quiet     --no-warn-script-location --index-url\
           \ https://pypi.org/simple --trusted-host https://pypi.org/simple 'yapf'\
-          \ 'kfp==2.0.0-beta.8' && \"$0\" \"$@\"\n"
+          \ 'kfp==2.0.0-beta.12' && \"$0\" \"$@\"\n"
```

Comment:
Can we avoid the diff for files that aren't affected by this fix? This diff just a version bump.

Reply:
I think all those changed when i auto generated the golden snapshots